src/readme.html

```diff
@@ -80,12 +80,12 @@ <h2 id="resources">Online Resources</h2>
 	<dt><a href="https://docs.classicpress.net/">ClassicPress Documentation</a></dt>
 		<dd>Here you'll find up-to-date reference information about ClassicPress.</dd>
 	<dt><a href="https://www.classicpress.net/blog/">ClassicPress Blog</a></dt>
-		<dd>This is where you&#8217;ll find the latest updates and news related to ClassicPress. Recent ClassicPress news appears in your administrative dashboard by default.</dd>
+		<dd>This is where you&#8217;ll find the latest updates and news related to ClassicPress as well as tutorials and other interesting snippets of information. Recent ClassicPress news appears in your administrative dashboard by default.</dd>
 	<dt><a href="https://forums.classicpress.net/c/support">ClassicPress Support Forums</a></dt>
 		<dd>The support forums are a great way to get help from other ClassicPress users. Many community members and ClassicPress leaders are active here.</dd>
 	<dt><a href="https://www.classicpress.net/join-slack/">ClassicPress Slack</a></dt>
 		<dd>There is an active online chat channel that is used for discussion among people who use ClassicPress and plan its further development.</dd>
-	<dt><a href="https://petitions.classicpress.net/">ClassicPress Petitions</a></dt>
+	<dt><a href="https://forums.classicpress.net/c/governance/petitions/77">ClassicPress Petitions</a></dt>
```

Comment:
Let's add a link to https://github.com/ClassicPress/subdomain-link for this.

Reply:
Resolved by ClassicPress/subdomain-link#3

Reply:
@bahiirwa do we want to add a link redirect for this?

```suggestion
	<dt><a href="https://link.classicpress.net/petitions">ClassicPress Petitions</a></dt>
```